lib/DDA/DDAPass.cpp

```diff
@@ -20,30 +21,6 @@ using namespace SVFUtil;
 
 char DDAPass::ID = 0;
 
-static llvm::cl::opt<unsigned> maxPathLen("maxpath",  llvm::cl::init(100000),
-        llvm::cl::desc("Maximum path limit for DDA"));
-
-static llvm::cl::opt<unsigned> maxContextLen("maxcxt",  llvm::cl::init(3),
-        llvm::cl::desc("Maximum context limit for DDA"));
-
-static llvm::cl::opt<string> userInputQuery("query",  llvm::cl::init("all"),
-        llvm::cl::desc("Please specify queries by inputing their pointer ids"));
-
-static llvm::cl::opt<bool> insenRecur("inrecur", llvm::cl::init(false),
-                                      llvm::cl::desc("Mark context insensitive SVFG edges due to function recursions"));
-
-static llvm::cl::opt<bool> insenCycle("incycle", llvm::cl::init(false),
-                                      llvm::cl::desc("Mark context insensitive SVFG edges due to value-flow cycles"));
-
-static llvm::cl::opt<bool> printCPts("cpts", llvm::cl::init(false),
-                                     llvm::cl::desc("Dump conditional points-to set "));
-
-static llvm::cl::opt<bool> printQueryPts("print-query-pts", llvm::cl::init(false),
-        llvm::cl::desc("Dump queries' conditional points-to set "));
-
-static llvm::cl::opt<bool> WPANUM("wpanum", llvm::cl::init(false),
-                                  llvm::cl::desc("collect WPA FS number only "));
-
```

Comment:
/// register this into alias analysis group
//static RegisterAnalysisGroup<AliasAnalysis> AA_GROUP(DDAPA);
static llvm::cl::bits<PointerAnalysis::PTATY> DDASelected(llvm::cl::desc("Select pointer analysis"),
        llvm::cl::values(
            clEnumValN(PointerAnalysis::FlowS_DDA, "dfs", "Demand-driven flow sensitive analysis"),
            clEnumValN(PointerAnalysis::Cxt_DDA, "cxt", "Demand-driven context- flow- sensitive analysis")
        ));


The above should also be moved to options.cpp